### DISCUSSION TITLE
Linux Automated Setup Guide -> Anaconda doesn't set up LSD

### DISCUSSION THREAD

**Discussion by drangen-developer** [hlky/stable-diffusion-webui#757] [Q&A]
During step 6 it says "Step 6: The script will then proceed to generate the Conda environment for the project. It will be created using the name lsd (not a mistake ;) ), which is short for Linux Stable Diffusion."
However, it didn't do that in my case. conda info --envs gives out:
```
# conda environments:
#
base                  *  /home/***/anaconda3
```

Which in turn makes the installation fail:
```
Running linux-setup.sh...

EnvironmentNameNotFound: Could not find conda environment: lsd
You can list all discoverable environments with `conda info --envs`.


python: can't open file '/home/***/StableDiffusion/ultimate-stable-diffusion/scripts/relauncher.py': [Errno 2] No such file or directory

```
It's likely a result if me having screwed up the setting up of the Anaconda environment, but I'm not sure what to exactly do there.
Thanks in advance.

**Comment by cstueckrath** [comment]
did you run the script from (base)? -> conda activate

**Reply by drangen-developer** [reply]
> did you run the script from (base)? -> conda activate

Could you elaborate, I'm not very familiar with Anaconda.
Before the identifier, the terminal does display (base), if that is what you mean.

**Reply by cstueckrath** [reply]
did you run the newest version of the script (with the new repo?)
Should state "# Version: 1.8.1" in line 5

**Reply by drangen-developer** [reply]
Ah yeah, that seemed to be it.
Thanks

**Reply by drangen-developer** [reply]
On testing, it wasn't nevermind, seems to have been a little hasty there.
It has changed into this though:
```
Starting Ultimate Stable Diffusion using previous parameters. Please wait...
Running linux-setup.sh...

EnvironmentNameNotFound: Could not find conda environment: lsd
You can list all discoverable environments with `conda info --envs`.


Relauncher: Launching...
Traceback (most recent call last):
  File "/home/hagius/StableDiffusion/stable-diffusion-webui/scripts/webui.py", line 3, in <module>
    from frontend.frontend import draw_gradio_ui
ModuleNotFoundError: No module named 'frontend'
Relauncher: Process is ending. Relaunching in 1s...
	Relaunch count: 1
```

**Reply by cstueckrath** [reply]
what happens when you manually create the env?
`conda env create -f environment.yaml`

**Reply by drangen-developer** [reply]
```
conda env create -f environment.yaml

EnvironmentFileNotFound: '/home/hagius/environment.yaml' file not found
```
This kind of exception

**Reply by cstueckrath** [reply]
no, you have to clone the repo first of course or at least point to the one downloaded by the script.

You should have a stable-diffusion-webui folder right where your script was.
so go to /home/hagius/StableDiffusion/stable-diffusion-webui and run it from there
